packages/typescript-estree/src/convert.ts

```diff
@@ -1821,10 +1837,13 @@ export class Converter {
       case SyntaxKind.MetaProperty: {
         return this.createNode<TSESTree.MetaProperty>(node, {
           type: AST_NODE_TYPES.MetaProperty,
-          meta: this.createNode<TSESTree.Identifier>(node.getFirstToken()!, {
-            type: AST_NODE_TYPES.Identifier,
-            name: getTextForTokenKind(node.keywordToken),
-          }),
+          meta: this.createNode<TSESTree.Identifier>(
+            node.getFirstToken()! as any,
```

Comment:
shouldn't this be `ts.Token<SyntaxKind.NewKeyword | SyntaxKind.ImportKeyword>`? Is there a way to make this work without the any?

Reply:
i have idea how to solve this :>

Reply:
worst case if it doesn't work, you can remove the non-null assertion.

```suggestion
            node.getFirstToken() as any,
```

Reply:
i had to to bigger refactoring but its working.

Reply:
```suggestion
            node.getFirstToken()! as ts.Token<typeof node.keywordToken>,
```